### DISCUSSION TITLE
know which phone mdel

### DISCUSSION THREAD

**Discussion by ridharezzag** [judemanutd/AutoStarter#72] [Ideas]
it will be nice if you add a function that return which phone brand is, based on this we can provide a link to user to explain to him how to activate autostart, just a suggestion, thank you

**Comment by judemanutd** [comment]
Thanks for the suggestion @ridharezzag . You should be able to achieve this using `Build.BRAND.toLowerCase(Locale.getDefault())`. It is an inbuilt function so I didn't see the point in duplicating that functionality in the library. Is there a particular reason you want the library to provide you with that data rather than retrieving it within your own app?

**Reply by ridharezzag** [reply]
Thanks for the answer, actually no big reason for this request, it was just a suggestion, since the library already run that function I said maybe It's good to return the value, that may save us another run for the function, but if you want to keep the library small as you mentioned in the README, you are right you are saving as some extra few lines of code, thanks for this library I was doing the work by myself but now because of this library you're saving me time by doing all the work for us and gathering all the packages in one place thank you again

**Reply by judemanutd** [reply]
Always glad to hear that though the credit isn't mine, I based alot of this off of stackoverflow. 

Anyways closing this request for now mainly because a feature request like this could spiral out very quickly into others that end up either duplicating the functionality that already exist or changing it into something entirely different.
